tests/installation/test_installer.py

```diff
@@ -1831,3 +1832,28 @@ def test_installer_can_handle_old_lock_files(
 
     # colorama will be added
     assert 8 == installer.executor.installations_count
+
+
+def test_quiet(package, pool, locker, env, installed, config, repo):
```

Comment:
```suggestion
@pytest.mark.parameterize("quiet,lines", [(True, []), (False, [".."])]
def test_quiet(package, pool, locker, env, installed, config, repo, quiet, lines):
```

Would be good to test both the cases here even thought this is indirectly verified elsewhere.

Reply:
I updated this PR with the suggested changes. In the `quiet=False` assertion, I tested with the whole command output but I'm not sure if this is the proper way. I don't know a better way to compare the output, do you have any suggestions?